src/v3/src/components/Select/Select.tsx

```diff
@@ -47,6 +48,19 @@ const Select: UISchemaElementComponent<UISchemaElementComponentWithValidationPro
   const focusRef = useAutoFocus<HTMLSelectElement>(focus);
   const hasErrors = typeof errors !== 'undefined';
 
+  const getOptions = (): IdxOption[] | undefined => {
+    if (Array.isArray(customOptions) || Array.isArray(options)) {
+      return customOptions ?? options;
+    }
+
+    if (typeof options === 'object') {
+      return Object.entries(options)
+        .filter(([key, val]) => key !== '' && val !== '')
+        .map(([key, val]) => ({ label: val, value: key } as IdxOption));
+    }
+    return undefined;
+  };
```

Comment:
I pulled this from https://github.com/okta/okta-signin-widget/pull/3110 because of the update I made to the renderers tester function for Select boxes (which was also pulled from that PR). I wanted to avoid future merge conflicts which is why I pulled that here (assuming this will get merged first).